.github/workflows/post-release.yml

```diff
@@ -33,6 +33,7 @@ jobs:
         cat notes-${{ env.RELEASE_VERSION }}.md
     - name: Create GitHub release
       id: release
+      # Unmaintained: https://github.com/actions/create-release
```

Comment:
Whats the point of this comment?  We have Issues for our backlog and PRs for resolving things.